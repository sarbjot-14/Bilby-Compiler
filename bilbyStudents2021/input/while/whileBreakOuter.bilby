main{
	mut count := 0;
	mut count2 := 0;
	while (count < 5) {
		print "outer" $n;
		print count $n;
		count := count +1;
		while (count2 < 7) {
			print "inner" $n;
			print count2 $n;
			count2 := count2 +1;
			if(count2 ==4){
				print "FOURR" $n;
				break;
			}
		}
		count2 := 0;
		
		
	}
}