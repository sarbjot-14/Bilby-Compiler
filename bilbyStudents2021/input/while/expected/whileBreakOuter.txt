outer 0
inner 0
inner 1
inner 2
inner 3
FOURR
outer 1
inner 0
inner 1
inner 2
inner 3
FOURR
outer 2
inner 0
inner 1
inner 2
inner 3
FOURR
outer 3
inner 0
inner 1
inner 2
inner 3
FOURR
outer 4
inner 0
inner 1
inner 2
inner 3
FOURR
